tests/test_modeling_common.py

```diff
@@ -1327,16 +1328,30 @@ def _create_and_check_torch_fx_tracing(self, config, inputs_dict, output_loss=Fa
                             (past_mask, inputs_to_test[1]["attention_mask"]), dim=1
                         )
 
+            if "inputs_embeds" in inspect.signature(model.forward).parameters:
```

Comment:
Won't this clash with `input_ids` in the inputs?

Reply:
No as we add a new set of inputs to test, independent of the previous that uses `input_ids`.

Reply:
My understanding of the test set-up is that the inputs to the model will now include both input_ids and input_embeds, which shouldn't be the case.

Reply:
No, `inputs_to_test` is a list of dict, with each dict being one input to the model.